docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -161,6 +162,27 @@ image::images/alert-change-status.png[Bulk action menu with multiple alerts sele
 
 * In an alert's details flyout, click *Take action* and select a status.
 
+[float]
+[[apply-alert-tags]]
+==== Apply and filter alert tags
+
+Use alert tags to organize related alerts into categories that you can filter and group. For example, use the `False Positive` alert tag to label a group of alerts as false positives. Then, search for them by entering the `kibana.alert.workflow_tags : "False Positive"` query into the KQL bar.  
+
+NOTE: You can manage alert tag options by updating the `securitySolution:alertTags` advanced setting. Refer to <<manage-alert-tags>> for more information. 
+
+To apply or remove alert tags on individual alerts, do one of the following:
+
+** In the Alerts table, click *More actions* (*...*) in the alert's row, then click *Apply alert tags*. Select or unselect tags, then click *Apply tags*. 
+** In an alert’s details flyout, click *Take action -> Apply alert tags*. Select or unselect tags, then click *Apply tags*. 
+
+To apply or remove alert tags on multiple alerts, select the alerts you want to change, then click *Selected _x_ alerts* at the upper-left above the table. Click *Apply alert tags*, select or unselect tags, then click *Apply tags*. 
+
+
+[role="screenshot"]
+image::images/bulk-apply-alert-tag.png[Bulk action menu with multiple alerts selected, 450]
+
+TIP: Add the `kibana.alert.workflow_tags` field to the Alerts table displays alert tags. Refer to <<customize-the-alerts-table>> to learn more about adding fields to the Alerts table. 
```

Comment:
```suggestion
TIP: Add the `kibana.alert.workflow_tags` field so that the Alerts table displays alert tags. Refer to <<customize-the-alerts-table>> to learn more about adding fields to the Alerts table. 
```

Reply:
Hm, I'm torn on this edit because I think your suggestion reads better, but I think the original is more direct and clear. Do you think it's safe to assume that users will understand that they need to modify the Alerts table to display specific fields?

Reply:
This is a good compromise, @joepeeples - thanks both!